client/header/activity-panel/activity-header/style.scss

```diff
@@ -29,10 +29,13 @@
 }
 
 .woocommerce-layout__inbox-panel-header {
-	padding: 15px 43px 10px;
-	h3 {
-		margin: 5px 0;
-		font-weight: 500;
+	padding: $gap * 2 42px $gap;
```

Comment:
Sorry to dump this on you, but I think we could actually migrate the styles you made in `homescreen/style.scss` here and just add some margin around the inbox panel.

```scss
#activity-panel-inbox {
	margin: 0 20px;
}
```

This way we can keep the base styles the same and don't need to calculate/override padding differences each time we use this component.